reference/service/collaboration.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Collaboration
+---
+
+The Pulumi Cloud Console offers multiple that people can collaborate on a
```

Comment:
"offers multiple that" .... missing a word in here.

Reply:
Fixed, "allows multiple ways for people to collaborate on a stack"